src/psyclone/tests/dynamo0p3_builtins_test.py

```diff
@@ -292,150 +354,250 @@ def test_dynbuiltfactory_str():
     assert "Factory for a call to a Dynamo built-in" in str(factory)
 
 
-def mesh_code_present(code):
-    '''this test checks for the existance of mesh code. This exists for
-    all builtins with dm = True (although it is not actually required!) so
-    each test can call this function'''
-    assert "      USE mesh_mod, ONLY: mesh_type" in code
-    assert "      TYPE(mesh_type), pointer :: mesh => null()" in code
-    output_dm_1 = (
-        "      !\n"
-        "      ! Create a mesh object\n"
-        "      !\n"
-        "      mesh => f1%get_mesh()\n"
-        "      !\n")
-    print output_dm_1
-    assert output_dm_1 in code
+# ------------- Adding (scaled) fields ------------------------------------- #
 
 
-def test_builtin_set_str():
-    ''' Check that the str method of DynSetFieldScalarKern returns the
-    expected string '''
+def test_X_plus_Y():     # Z = X + Y
```

Comment:
Please could you 'fix' the pylint warnings about invalid function names by adding a `  # pylint: disable=invalid-name` after each culprit.

Reply:
Disabled pylint warnings as requested. This also led to deletion of the inline comments after built-in tests' definitions as the functionality is described in detail in each test anyway.